src/index.html

```diff
@@ -1,13 +1,51 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image-container">
+        <img
+          src="./images/imac.jpeg"
+          alt="card-image"
+          class="card__image"
+        >
+      </div>
+      <h1 class="card__header header-position">
```

Comment:
remove all such classes, add margin-down directly to this class

```suggestion
      <h1 class="card__header">
```